docs/content/en/docs/reference/clusterspec/cni.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "cni plugin configuration"
+linkTitle: "cni"
+weight: 20
+description: >
+EKS Anywhere cluster yaml cni plugin specification reference
+---
+
+### Specifying CNI Plugin in EKS Anywhere cluster spec
+
+EKS Anywhere currently supports two CNI plugins: Cilium and Kindnet. Up until the 0.7.x releases,
+the plugin had to be specified using the `cni` field on cluster spec.
+Starting release 0.8, the plugin should be specified using the new `cniConfig` field as follows:
+
+1. For selecting Cilium as the CNI plugin:
```

Comment:
```suggestion
* For selecting Cilium as the CNI plugin:
```
After the bullet, I'd suggest indenting all the lines in the yaml file to align with the bulleted text.